arguments-optional.js

```diff
@@ -0,0 +1,22 @@
+/**
+ *sums two arguments together. If only one argument is provided, returns a function that expects one argument and returns the sum. If either argument isn't a valid number, returns undefined.
+ *
+ * @param {*} a
```

Comment:
```js
@param {number} a number to add
@param {number} b number to add
```

Reply:
the parameter can be any type, not just a number.

Reply:
than `@param{any}`

Reply:
<img width="449" alt="screen shot 2018-07-15 at 6 37 57 pm" src="https://user-images.githubusercontent.com/26444164/42743304-4b6cf7f2-885e-11e8-8ff7-21e81798bb1b.png">

going by the documentation..

Reply:
http://usejsdoc.org/tags-param.html

Look at you digging into the documentation calling my bullshit! Great job! +100 points